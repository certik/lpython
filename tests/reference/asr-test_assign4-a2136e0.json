{
    "basename": "asr-test_assign4-a2136e0",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_assign4.py",
    "infile_hash": "561fdb2fa9ddab21de687217119c385589018aeb839f53ad9ece0942",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_assign4-a2136e0.stderr",
    "stderr_hash": "b47803e86e3b43c3191ec97fe750e85473c6a1f30918d97a1d0b58b3",
    "returncode": 2
}