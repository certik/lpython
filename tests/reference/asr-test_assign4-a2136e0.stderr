semantic error: Type mismatch in assignment, the types must be compatible
 --> tests/errors/test_assign4.py:5:5
  |
5 |     f = x
  |     ^   ^ type mismatch ('f64' and 'i32')
